eml/screens/lectures/CompSwipeScreen.jsx

```diff
@@ -65,11 +65,7 @@ export default function LectureSwipeScreen({ route }) {
     swiperRef.current.scrollBy(1, true);
     setScrollEnabled(true);
 
-    if (index === combinedLecturesAndExercises.length - 1) {
-      return true;
-    } 
-
-    return false;
+    return index === combinedLecturesAndExercises.length - 1;
   };
```

Comment:
I am not sure about this. Why would you change it?

Reply:
Like in this https://github.com/Educado-App/educado-mobile/pull/221#discussion_r1410319842, it is redundant to say if true then true, if false then false.